dev/io.openliberty.restfulWS.internal.logging.filter/bnd.bnd

```diff
@@ -0,0 +1,46 @@
+#*******************************************************************************
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.restfulWS.internal.logging.filter
+
+app-resources= \
+  META-INF/services/javax.ws.rs.ext.Providers
+
+Service-Component: \
```

Comment:
This is more of an education question, but can you explain what this service component does?

Reply:
This is basically the same thing as what `@Component` does in the code - it sets up an OSGi Declarative Service.  But this is a unique one in that it is actually setting up a DS component for another bundle (the classloading service bundle) - so it cannot be created using annotations.  This is what makes it possible for user applications to load resource files (like META-INF/services/... files) from Liberty bundles.